lib/package-lock.json

```diff
@@ -1,461 +1,8 @@
 {
   "name": "@replit/protocol",
   "version": "0.2.29",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
can this file be kept as-is? (i'm guessing that somebody had a newer version of node)